beater/api/mux.go

```diff
@@ -264,7 +264,6 @@ func apmMiddleware(m map[request.ResultID]*monitoring.Int) []middleware.Middlewa
 		middleware.TimeoutMiddleware(),
 		middleware.RecoverPanicMiddleware(),
 		middleware.MonitoringMiddleware(m),
-		middleware.RequestTimeMiddleware(),
```

Comment:
Were we resorting to this middleware because previously the APMEvent didn't have a timestamp we could rely on for the timings?

Reply:
The event struct has always had a timestamp.

There was nothing preventing us from setting the timestamp on the request context before, we (I think I?) just chose to store it in context instead, probably because middleware seemed like the path of least resistance. Storing it directly on the context is simpler.